platform/jobs/edxPlatformAccessibilityMaster.groovy

```diff
@@ -130,7 +130,9 @@ secretMap.each { jobConfigs ->
            predefinedPropsMap.put('TARGET_URL', JENKINS_PUBLIC_BASE_URL +
                                   'view/accessibility/job/' + jobConfig['jobName'] + '/${BUILD_NUMBER}/')
            downstreamParameterized JENKINS_PUBLIC_GITHUB_STATUS_PENDING.call(predefinedPropsMap)
-           shell("cd ${jobConfig['repoName']}; RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh")
+           shell("cd ${jobConfig['repoName']}; " +
+                 'RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh; ' +
+                 "pip freeze > \${WORKSPACE}/${jobConfig['repoName']}/test_root/log/pip_freeze.log")
```

Comment:
If ./scripts/accessibility-tests.sh exits with an error code and then the pip freeze exits successfully, will the build pass or fail?